### PR TITLE
Relax requirements in eigen22d test. Adapt result from transformPlane test based on used Eigen version. 

### PR DESCRIPTION
**Pull request by SergioRAgostinho**

### CODE REVIEW

**Comment by SergioRAgostinho** [comment]
I just discovered the eigen22f test is still failing on my 18.04 environment. Gonna have a look at it again although I'm fearing this is gonna get us back to the original problems in #1702.

**Comment by SergioRAgostinho** [comment]
This one is also good to go.

File: test/common/test_eigen.cpp [hunk 1]
```diff
@@ -932,6 +932,13 @@ TEST (PCL, transformPlane)
   transformationd.linear() = (Eigen::Matrix3d) Eigen::AngleAxisd(M_PI/7, Eigen::Vector3d::UnitY())
   * Eigen::AngleAxisd(M_PI/4, Eigen::Vector3d::UnitZ());
   test << 5.35315, 2.89914, 0.196848, -49.2788;
+
+  // From Eigen 3.3.2 onwards, the Eigen::Hyperplane::transform always returns
+  // coefficients with the normal normalized
+  #if EIGEN_VERSION_AT_LEAST (3, 3, 2)
+    test /= test.head<3> ().norm ();
```
Comment (taketwo) [inline test/common/test_eigen.cpp]: Shouldn't we rather tweak the function to return consistent results no matter what the Eigen version is? That is, to return normalized normal?

**Comment by taketwo** [comment]
Thanks for the ping.

**Pull request merged**